pyatlan/generator/templates/methods/attribute/a_p_i_field.jinja2

```diff
@@ -0,0 +1,86 @@
+
+        @classmethod
+        @init_guid
+        def create(
+            cls, 
+            *, 
+            name: str, 
+            parent_api_object_qualified_name: str = None, 
+            parent_api_query_qualified_name: str = None, 
+            connection_qualified_name: Optional[str] = None, 
+            api_field_type: Optional[str] = None,
+            api_field_type_secondary: Optional[str] = None,
+            is_api_object_reference: Optional[bool] = False,
+            reference_api_object_qualified_name: Optional[str] = None,
+            api_query_param_type: Optional[APIQueryParamTypeEnum] = None,
+        ) -> APIField.Attributes:
```

Comment:
Should be `creator()` (ref: https://github.com/atlanhq/atlan-python/pull/403/files#r1800760792)